Packs/BranddefenseCompleteIntegration/Integrations/BranddefenseCompleteIntegration/BranddefenseCompleteIntegration.yml

```diff
@@ -0,0 +1,202 @@
+  - arguments:
+    - name: audit_log_id
+      required: true
+    name: branddefense_get_specific_audit_log
+    description: Get a specific audit log from Brandefense platform
+    outputs:
+    - contextPath: branddefense_requested_audit_log
+  - arguments:
+    - name: search_uuid
+    name: branddefense_get_threat_search
+    description: Start a threat search on Brandefense
+  - arguments:
+    - name: incident_code
+      required: true
+    name: branddefense_get_specific_incident
+    description: Search for a specific incident on Brandefense
+    outputs:
+    - contextPath: branddefense_requested_incident
+  - arguments:
+    - description: The required code parameter should be given to perform incident searches on Brandefense.
+      name: incident_code
+      required: true
+    - description: 'The required status parameter should be given to perform change incident status Possible Values: ''OPEN'' ''IN_PROGRESS'' ''CLOSED'' ''RISK_ACCEPTED'' ''REJECTED'''
+      isArray: true
+      name: incident_status
+      required: true
+    name: branddefense_change_incident_status
+    description: Change the status of the incident on Brandefense
+  - arguments:
+    - description: The required code parameter should be given to perform incident searches on Brandefense.
+      name: incident_code
+      required: true
+    name: branddefense_get_incident_indicators
+    description: Get incident indicators from Brandefense
+    outputs:
+    - contextPath: branddefense_requested_incident_indicators
+  - arguments:
+    - name: indicator_type
+      required: true
+    name: branddefense_get_indicators
+    description: Get indictors from Brandefense
+    outputs:
+    - contextPath: branddefense_requested_indicator
+  - arguments:
+    - description: The required ioc_type parameter allows you to select Indicators of Compromise (IoCs) types from the threat lists.
+      name: ioc_type
+      required: true
+    name: branddefense_get_ioc
+    description: Get IOCs from Brandefense
+    outputs:
+    - contextPath: branddefense_ioc
+      description: hash,domain,url veya ip address girilmeli
+  - arguments:
+    - description: Cti Rule Created at range
+      name: created_at__range
+    - description: Cti rule search
+      name: search
+    - name: tag
+    - name: source__ilike
+    name: branddefense_get_cti_rules
+    description: Get CTI rules from Brandefense
+  - arguments:
+    - name: value
+    description: Create Threat Search endpoint allows you to post indicator of compromises to investigate suspicious/malicious IP, Domain, URL address or Hash values.
+    name: branddefense_create_threat_search
+    outputs:
+    - contextPath: search_result
+  - arguments:
+    - name: created_at
+    name: branddefense_get_incidents
+    description: Get all incidents from Brandefense
+    outputs:
+    - contextPath: branddefense_all_incidents
+  dockerimage: demisto/python3:3.10.13.74666
+  isfetch: true
```

Comment:
Does this integration fetches incidents? if so, please add a `fetch_incidents `command using this guide https://xsoar.pan.dev/docs/integrations/code-conventions#fetch-incidents